html/pfappserver/root/admin/nodes.tt

```diff
@@ -188,10 +188,108 @@ form {
                     <input type="hidden" name="searches.0.value" value="on"/>
                 </form>
               </li>
+
+              <li class="nav-header">[% l('Custom Predefined Search') %]</li>
+              <div class="accordion" id="accordion-SwithGroup">
```

Comment:
There is no indicator to tell if there are some sub menu

![image](https://cloud.githubusercontent.com/assets/2727473/15577447/99eaf118-232a-11e6-95c2-1d4dd1742fe2.png)